onnxruntime/python/tools/transformers/float16.py

```diff
@@ -436,6 +437,7 @@ def convert_float_to_float16(
                     node.input[i] = output_name
                     break
 
+    accuracy_type = TensorProto.FLOAT16 if use_bfloat16_as_blocked_nodes_dtype else TensorProto.FLOAT
```

Comment:
This change is not generic. Instead, let's try allow user to pass block list configuration for (operator domain, operator name, input index, data type).

Reply:
I think it's a rare case that different ops be cast to different accT(like some Attention to fp32 and some Attention to bfp16). we can enhance it when there is a need.

Reply:
according to name, shall the accuracy type be TensorProto.**BFLOAT16**?

Reply:
fixed